src/components/Content.js

```diff
@@ -1,52 +1,25 @@
-import React, {useState} from 'react';
+import React, { Fragment } from 'react';
+import {Container, Row, ListGroup, ListGroupItem} from "react-bootstrap";
 import './Content.css';
-import Iframe from "react-iframe";
-import Spinner from 'react-spinkit';
```

Comment:
Is the loading icon gone completely? I suppose it's not needed anyway. But let me know.